Podfile

```diff
@@ -428,4 +428,16 @@ post_install do |installer|
   yellow_marker = "\033[33m"
   reset_marker = "\033[0m"
   puts "#{yellow_marker}The abstract target warning below is expected. Feel free to ignore it.#{reset_marker}"
+
+  # Fix a compiling issue in Xcode 15 beta.
+  # This line in particular https://github.com/mrackwitz/MRProgress/blob/0.8.3/src/Components/MRProgressOverlayView.m#L811
+  # The error message is "Multiple methods named 'setProgress:' found with mismatched result, parameter type or attributes"
+  # We are going to replace the `id` with `MRProgressView *` (or any other type that has a method `setProgress:` that takes a float argument).
+  mrprogress_filepath = "#{project_root}/Pods/MRProgress/src/Components/MRProgressOverlayView.m"
+  File.chmod(0o644, mrprogress_filepath)
+  lines = File.readlines(mrprogress_filepath)
+  if lines[810] == "        [((id)self.modeView) setProgress:self.progress];\n"
+    lines[810] = "        [((MRProgressView *)self.modeView) setProgress:self.progress];\n"
+    File.write(mrprogress_filepath, lines.join)
+  end
```

Comment:
This is a nifty way of patching the issue 🤓 

Have you considered forking the repo? Might set us up for addressing future issues.

Reply:
That didn't cross my mind. 🥲 

But I did think about removing this library, because it's only used in one single place: showing an activity indicator during uploading a featured image to a post's settings. I got lazy and decided to make this one line change instead. 😸

Reply:
😄 

I came here to write the same thing. I found that same as you did (_sigh of relief_).

If we go in with the intention of removing this library, then I'd say it's okay to merge this approach. I wonder if we could get the help of some iOS UI dev 🤓